HW1/P4/P4.py

```diff
@@ -0,0 +1,44 @@
+from pyspark import SparkContext
+sc = SparkContext("local[8]")
+import numpy as np
+import csv
+from P4_bfs import *
+
+def quiet_logs(sc):
+    logger = sc._jvm.org.apache.log4j
+    logger.LogManager.getLogger("org").setLevel(logger.Level.WARN)
+    logger.LogManager.getLogger("akka").setLevel(logger.Level.WARN)
+    logger.LogManager.getLogger("amazonaws").setLevel(logger.Level.WARN)
+
+quiet_logs(sc)
+
+def combine_characters(x):
```

Comment:
Really nice that you are documenting your code, minor issue would be good to clearly mention what input it takes in the docs so it is easier to reuse later.
Also this is part of the standard library - https://docs.python.org/2/library/itertools.html#itertools.permutations